malaria-ios/ViewControllers/UserProfilePillTableViewCell.swift

```diff
@@ -0,0 +1,48 @@
+import UIKit
+import DoneToolbarSwift
+
+// MARK: - Table View Cell
+
+class UserProfilePillTableViewCell: UITableViewCell {
+  
+  @IBOutlet weak var pillName: UILabel!
+  @IBOutlet weak var quantityTextField: UITextField!
+  
+  weak var delegate : SaveRemainingPillsProtocol?
+  
+  var toolBar: ToolbarWithDone!
+  
+  override func layoutSubviews() {
```

Comment:
Why layoutSubviews? I believe this is called when it is being rendered therefore, the toolbar would be instantiated here as well continuously. Try to use the init method instead.